.gitignore

```diff
@@ -40,3 +40,5 @@ build
 
 # Environment files
 *.env
+Config.toml
+examples/**/Dependencies.toml
```

Comment:
Can we add `*.deb` here as well. Unless it might fail the publish release workflow.
https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/actions/runs/7984743086/job/21804693645